server/controllers/caseNotesController.ts

```diff
@@ -65,13 +60,14 @@ export default class CaseNotesController {
       const prisonApiClient = this.prisonApiClientBuilder(clientToken)
       const [caseNotesUsage, caseNotesPageData, inmateDetail] = await Promise.all([
         prisonApiClient.getCaseNotesUsage(req.params.prisonerNumber),
-        this.caseNotesService.get(
-          userToken,
+        this.caseNotesService.get({
+          token: clientToken,
```

Comment:
We now use the client token for the GET request, passing in the includeSensitive=true query parameter when the user has role POM